README.md

```diff
@@ -21,21 +21,21 @@ dcos-metrics collects, tags, and transmits metrics from every node, container, a
 
 * [DataDog][plugin-datadog]
 * [Librato][plugin-librato]
-* [Prometheus][plugin-prometheus] (DC/OS 1.9 and 1.10 only)
+* [Prometheus][plugin-prometheus] (DC/OS 1.9+)
```

Comment:
This plugin is for 1.9 and 1.10 only - 1.11 incorporates its functionality into dcos-metrics